newt_install.sh

```diff
@@ -34,7 +34,12 @@ done
 
 rm -rf $GOPATH/bin $GOPATH/pkg
 
-go install mynewt.apache.org/newt/newt
-[[ $? -ne 0 ]] && exit 1
+for i in 1 2 3
+do
+   go install mynewt.apache.org/newt/newt
+   [[ $? -eq 0 ]] && break
+   [[ $i -eq 3 ]] && exit 1
+   sleep 30
+done
```

Comment:
I would think that `go install` only installs what is already download by `go get` but I am not sure. Anyway I suggest some refactoring to make this block into a function like this:

```
retry_3_times() {
    for i in 1 2 3; do
        $*
        [[ $? -eq 0 ]] && break
        [[ $i -eq 3 ]] && exit 1
        sleep 30
    done
}
```

Then you can call it like:

`retry_3_times go install mynewt.apache.org/newt/newt`